ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/ClusterStatusResource.java

```diff
@@ -47,6 +64,51 @@ public Response checkClusterStatus() {
   }
 
   private ClusterStatusResponse getResponse() {
-    return new ClusterStatusResponse(heartbeatAgent.getHostsStatus());
+    final Map<HostInfo, HostStatus> allHostStatus = heartbeatAgent.getHostsStatus();
+
+    final Map<HostInfoEntity, HostStatusEntity> response = allHostStatus
+        .entrySet()
+        .stream()
+        .collect(Collectors.toMap(
+            entry -> new HostInfoEntity(entry.getKey().host(), entry.getKey().port()) ,
+            entry -> new HostStatusEntity(entry.getValue().isHostAlive(),
+                                          entry.getValue().getLastStatusUpdateMs(),
```

Comment:
Since these are JSON entities, If they take the objects as arguments, I would need to write a serializer/deserializer which means they would depend on changes in the source. At least now, they are independent of additions of new fields.